.github/workflows/cli.yaml

```diff
@@ -41,3 +42,178 @@ jobs:
 
             - name: Build
               run: yarn build --filter @sunodo/cli
+
+    pack_upload_deb:
```

Comment:
Do we want to run these new jobs also on PR, or just on tag?
The workflow also runs on PRs.

Also I see there are 3 jobs, and quite some code duplication. Maybe we could all in one job?

Reply:
Just on tag I think. 
That's why I added `if: startsWith(github.ref, 'refs/tags/@sunodo/cli@')` section.

Reply:
For the duplications. I was thinking to use job template.

Reply:
I think we need a single job that runs only for the `on.push.tags` trigger.

Reply:
```
on:
  workflow_call:

```

Then use it in each jobs steps : 

```    uses:.//.github/workflows/common-cli.yml```

Reply:
It makes no sense to have each of these steps as independent Jobs since they won't need to run alone.

They are just steps that run every time, right?